### ISSUE TITLE
how to test a picture?

### ISSUE THREAD

**Issue by cmstudyscode** [ShuLiu1993/PANet#35]
Could you please add some information about test procedure????
i have run the infer_simple.py for visualize the outcome.
but here comes an error:
`Traceback (most recent call last):
  File "tools/infer_simple.py", line 176, in <module>
    main()
  File "tools/infer_simple.py", line 163, in main
    kp_thresh=2
  File "/home/cm/PANet/lib/utils/vis.py", line 183, in vis_one_image
    e.copy(), cv2.RETR_CCOMP, cv2.CHAIN_APPROX_NONE)
ValueError: not enough values to unpack (expected 3, got 2)
`
the order i used is:
`python tools/infer_simple.py --dataset coco2017 --cfg configs/panet/e2e_panet_R-50-FPN_2x_mask.yaml --load_ckpt panet_mask_step179999.pth --image_dir demo/sample_images --output_dir infer_outputs 
`
Can you help me!! thank u so much!

**Comment by rus8** [comment]
@cmstudyscode I faced the same problem. 
This error appears due to the small change in the OpenCV API. It seems that you installed opencv-python-4.1.0 via pip. [OpenCV 4.1.0 function cv2.findContours](https://docs.opencv.org/4.1.0/d3/dc0/group__imgproc__shape.html#gae4156f04053c44f886e387cff0ef6e08) returns only 2 values, while the PANet code was written for previous OpenCV versions which output 3 values.

**TL;DR:**
Change [line 182 in PANet/lib/utils/vis.py](https://github.com/ShuLiu1993/PANet/blob/f055f716a21896dab8907c46c12e216323baefdb/lib/utils/vis.py#L182) to: ` contour, hier = cv2.findContours(` by removing leading `_,`.
